src/main/java/com/avengers/gamera/controller/ArticleController.java

```diff
@@ -0,0 +1,31 @@
+package com.avengers.gamera.controller;
+
+import com.avengers.gamera.dto.article.ArticleGetDto;
+import com.avengers.gamera.dto.article.ArticlePostDto;
+import com.avengers.gamera.service.ArticleService;
+import io.swagger.v3.oas.annotations.Operation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("articles")
+@RequiredArgsConstructor
+@Validated
+public class ArticleController {
+    private final ArticleService articleService;
+
+    @PostMapping
+    @Operation(summary = "Create new article")
+    @ResponseStatus(HttpStatus.CREATED)
+    public ArticleGetDto createArticle(@Valid @RequestBody ArticlePostDto articlePostDto) {
```

Comment:
It seems like we don't need to use @Validated and @Valid in the mean time, please double check.

Reply:
fixed   (copy from user forgot to remove)